src/components/displaybook.js

```diff
@@ -1,13 +1,80 @@
-/* eslint-disable react/prop-types */
-export default function Book({ book }) {
+import React, { useEffect } from 'react';
+import '../styles/bookdisplay.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { getBookItems, removeBook } from '../redux/books/booksSlice';
+
+const Book = () => {
+  const dispatch = useDispatch();
+  const { books, isLoading, error } = useSelector((state) => state.books);
+
+  useEffect(() => {
+    dispatch(getBookItems());
+  }, [dispatch]);
+
+  const renderBook = (itemId, book) => {
+    const { category, title, author } = book;
+    return (
+      <div key={itemId} className="containerRender" style={{ display: 'flex' }}>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement, as your previous code reviewer asked for changes related to the design. You did a great job by adding the `circular` graph and those `vertical` dividers but I think you missed the `third` point. According to your add book, the form should be aligned properly with the rest of the design, Right now Your `form` is not aligned and your design looks inconsistent and unprofessional. Kindly keep in mind that the *design should match exactly with the `Zeplin` design*. Even the Senior Code Reviewer also mentioned this point.

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/ethred/book-store-React/assets/66476329/7036d535-6b05-4078-a286-55136c322836)

</td>
<td>

![Screenshot from 2023-09-02 23-53-35](https://github.com/ethred/book-store-React/assets/66476329/b14fbe8b-3539-48bf-be23-d40c566e70c2)


</td>
</tr>
</table>